spec/menu_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'menu'
+
+describe Menu do
+    it "can print the menu items" do
+        expect(subject.print_menu)
```

Comment:
Please add an expectation like `expect { subject.print_menu }.to output("...some menu here...").to_stdout`

Reply:
Ive added the menu items to this output statement.